Tests/Recurly/Subscription_Test.php

```diff
@@ -240,6 +240,11 @@ public function testUpdateNotes() {
 
     $notes = array("customer_notes" => "New Customer Notes", "terms_and_condititions" => "New Terms", "vat_reverse_charge_notes" => "New VAT Notes");
 
+    // You can also update custom fields through this endpoint
+    $cf = $subscription->custom_fields->offsetGet("shasta");
```

Comment:
Nit pick but I think the preferred style for this would be to just use the array operator: 
```php
$subscription->custom_fields["shasta"];
```

Reply:
Yeah, I think that's worth fixing. I'll push up a commit on top.

Reply:
I actually didn't know that the bracket syntax worked. TIL, thanks